tests/test_caching.py

```diff
@@ -455,10 +455,11 @@ def test_clear_cache(self, operate_on_empty_cache, nx=1000, ny=1000):
         for i in range(10):
             assert(len(_SymbolCache) == cache_size)
 
-            Function(name='u', grid=grid, space_order=2)
-            # Both u and u(inds) added to cache
-            assert(len(_SymbolCache) == cache_size + 2)
-
+            u = Function(name='u', grid=grid, space_order=2)
+            u._C_symbol
+            # All u, u(inds) and u.function through u._c_symbol added to cache
+            assert(len(_SymbolCache) == cache_size + 3)
+            del u
```

Comment:
what's this `del` for?

Reply:
same as test below force clear cache because its done at the end of the code block not start